indexing-service/src/main/java/org/apache/druid/indexing/input/DruidInputSource.java

```diff
@@ -484,6 +501,54 @@ private static SortedMap<WindowedSegmentId, Long> createWindowedSegmentIdFromTim
     return segmentSizeMap;
   }
 
+
+  public List<TimelineObjectHolder<String, DataSegment>> getLockedTimelineForInterval(
+      RetryPolicyFactory retryPolicyFactory,
+      String dataSource,
+      Interval interval
+  )
+  {
+    Preconditions.checkNotNull(interval);
+
+    // This call used to use the TaskActionClient, so for compatibility we use the same retry configuration
```

Comment:
This copied comment is invalid here as you _are_ using the `TaskActionClient` in this case.